### PR TITLE
Cmake: Fix issues with building netusb

### PR DESCRIPTION
**Pull request by finikorg**

### CODE REVIEW

File: subsys/usb/class/netusb/CMakeLists.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ zephyr_library_include_directories(
   )
 
 zephyr_library_sources_ifdef(
+  CONFIG_USB_DEVICE_NETWORK
```
Comment (SebastianBoe) [inline subsys/usb/class/netusb/CMakeLists.txt]: This should be removed. This entire directory is ifdef'd on CONFIG_USB_DEVICE_NETWORK
one level up.

add_subdirectory_ifdef(CONFIG_USB_DEVICE_NETWORK netusb)

**Review by SebastianBoe — CHANGES_REQUESTED** [review]

File: subsys/usb/class/netusb/CMakeLists.txt (hunk 1 above)
Comment (finikorg) [reply subsys/usb/class/netusb/CMakeLists.txt]: So how can I include netusb.c?

Comment (SebastianBoe) [reply subsys/usb/class/netusb/CMakeLists.txt]: zephyr_library_sources(
  netusb.c
  )

**Comment by jfischer-phytec-iot** [comment]
@loicpoulain Loic was faster, #4872

**Comment by finikorg** [comment]
Closing due to fixed in other PR

**Pull request closed without merging**